server/src/main/java/org/elasticsearch/action/get/GetRequest.java

```diff
@@ -109,9 +108,6 @@ public GetRequest(String index, String id) {
     @Override
     public ActionRequestValidationException validate() {
         ActionRequestValidationException validationException = super.validateNonNullIndex();
-        if (Strings.isEmpty(type)) {
```

Comment:
Do we need this change? I think the type should always be present, and a 'typeless' request is represented by using `_doc`.

Reply:
I guess we can change it later when we remove "_doc" as well. Thats the plan, isn't it?

Reply:
Right, we can eventually update these requests to not contain the `type` field at all.